src/libs/blueprint/conduit_blueprint_mesh.cpp

```diff
@@ -3174,6 +3349,64 @@ mesh::topology::unstructured::verify(const Node &topo,
                     chld_res &= verify_integer_field(protocol, chld, chld_info, "offsets");
                 }
 
+                // Polygonal & Polyhedral shape
+                if(chld.has_child("shape") && 
+                   chld["shape"].dtype().is_string() &&
+                   (chld["shape"].as_string() == "polygonal" || 
+                   chld["shape"].as_string() == "polyhedral"))
+                {
```

Comment:
I know that there was a problem in the original code here wherein some 10+ lines were duplicated between the single shape case and the mixed shape case, but persisting this problem to ~30 lines exacerbates it. Would it be possible to abstract this behavior into some form of helper function to cut down on the code duplication?

Reply:
Yes, I've added a helper function,`verify_poly_node()` that takes care of verifying for polygonal/polyhedral in the mixed and single shape case.